Doc/library/sqlite3.rst

```diff
@@ -1235,17 +1302,22 @@ Connection objects
 
    .. attribute:: isolation_level
 
-      This attribute controls the :ref:`transaction handling
-      <sqlite3-controlling-transactions>` performed by :mod:`!sqlite3`.
+      Legacy control of the :ref:`transaction handling
+      <sqlite3-transaction-control-isolation-level>` performed by :mod:`!sqlite3`.
       If set to ``None``, transactions are never implicitly opened.
       If set to one of ``"DEFERRED"``, ``"IMMEDIATE"``, or ``"EXCLUSIVE"``,
       corresponding to the underlying `SQLite transaction behaviour`_,
       implicit :ref:`transaction management
-      <sqlite3-controlling-transactions>` is performed.
+      <sqlite3-transaction-control-isolation-level>` is performed.
 
       If not overridden by the *isolation_level* parameter of :func:`connect`,
       the default is ``""``, which is an alias for ``"DEFERRED"``.
 
+      Using :attr:`autocommit` to control transaction handling is recommended
+      over using :attr:`!isolation_level`.
+      :attr:`!isolation_level` has no effect unless :attr:`autocommit` is set
+      to :data:`LEGACY_TRANSACTION_CONTROL`.
```

Comment:
```suggestion
      to :data:`LEGACY_TRANSACTION_CONTROL` (the default).
```

Again, clarify this is the default so users don't have to go look it up on their own.